### ISSUE TITLE
DatePicker shouldn't be announced as a table

### ISSUE THREAD

**Issue by eveleens** [OfficeDev/office-ui-fabric-react#10001]
## Codepen demonstrating the issue
 
https://developer.microsoft.com/en-us/fabric#/controls/web/datepicker

## Actual
Screen reader reading all the controls of calendar under table format as 6 row 5 column.

## Expected
Screen reader should not announce calendar controls under table format.It should read only days and date year information.

## Documentation describing expected behavior
MAS 1.3.1 - Info and Relationships

I'm not sure what we can do here, because the calendar is presented as table, but we got this filed from the accessibility team.

**Comment by marygans** [comment]
Hi @eveleens! Thanks for submitting this issue. I believe this issue is related to the existing issue #9641. If you agree, we can ping that open issue and get this one closed

**Comment by eveleens** [comment]
Hey @marygans - #9641 is saying that restructuring needs to happen to read the _correct_ rows, but this bug seems to say that columns shouldn't be read _at all_. 

Not sure what the expected behavior needs to be though.

**Comment by marygans** [comment]
Thanks @eveleens and my apologies for the delay.  @lorejoh12 do you have any insights into what the expected behavior should be based on the accessibility bug filed? 

My initial thinking was that this was intentional to provide more context to the user that the calendar is in a table format and they can move up, down, left, or right to span the calendar..

**Comment by lorejoh12** [comment]
The table structure was created before I took ownership of the control, so I can't speak to the reasoning that the semantics for announcing row/column were originally used. However, it is certainly convenient that we announce the table structure the way that we do, because it tells you about the navigation information, as well as announcing what row/column you're in only when it changes. For instance, if you're in week 30 (row header), and you use the right arrow to go to tomorrow, narrator doesn't have to announce that you're still in week 30, because the row didn't change.

We've gotten accessibility bugs from various teams before about issues with the table structure when there are bugs, but this is the first saying that the table structure should be eliminated entirely. MAS 1.3.1 doesn't really specify any clear reason that the table structure is wrong for what it's being used for as far as I know. If the accessibility team that filed your bug strongly disagrees, we can get them in contact with ours to come to a consensus on the correct semantics here, but otherwise I'd close this as "working as intended".
